zstacklib/zstacklib/utils/lv_metadata.py

```
@@ -0,0 +1,1010 @@
+            )
+
+            try:
+                with lvm.OperateLv(lv_path, shared=True):
+                    status = get_metadata_status(lv_path, lv_size)
+                if status.get('valid'):
+                    entry.schemaVersion = status.get('schema_version', '')
+                    entry.lastUpdateTime = status.get('last_update_time', 0)
+                    entry.vmName = status.get('vm_name', '')
+                    entry.architecture = status.get('architecture', '')
+                    entry.vmCategory = status.get('vm_category', '')
+            except Exception as e:
```

Ruff (0.15.7)

[warning] Do not catch blind exception: `Exception`

(BLE001)